### ISSUE TITLE
[Feature] Is there a process for visually adjusting the thresholds?

### ISSUE THREAD

**Issue by qsunyuan** [wandb/client#3624]
For example, I get the outputs (logits) of last layer of a segmentation neural network (eg: binary segmentation), and I get the probabilities with the sigmoid function. 

I pass it on the wandb webiste with wandb.Image(). 
 
Is there a way to adjust the threshold on the webpage to see different segmentation results?

**Comment by ramit-wandb** [comment]
Hi @qsunyuan,

Currently there is no such feature, but we do have an existing request for something like this. I'll increase the priority on the feature request for you!

Thanks,
Ramit

**Comment by qsunyuan** [comment]
Thx.
